### DISCUSSION TITLE
Welcome to bionicpython Discussions!

### DISCUSSION THREAD

**Discussion by nathfavour** [nathfavour/bionicpython#4] [Announcements]
Hi everyone interested in reading faster!

**Comment by shakemo** [comment]
Could you please share how I can install this, on macOS or linux?

**Comment by nathfavour** [comment]
Of course; I just included a setup.sh for MacOs and linux users!

All you need to do is clone the repo, and run the setup.sh and it will configure everything for you and let you know what to do next.

Thanks
